.github/workflows/pr-stale.yml

```
@@ -0,0 +1,23 @@
+    - cron: "0 0 * * *"
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - uses: actions/stale@v8f717f0dfca33b78d3c933452e42558e4456c8e7
+        with:
+          repo-token: ${{ steps.generate-token.outputs.token }}
```

actionlint (1.7.7)

property "generate-token" is not defined in object type {}

(expression)